--- a/debian/control    2020-06-12 13:50:07.005489917 -0400
+++ b/debian/control    2020-06-12 13:49:43.062442497 -0400
@@ -10,7 +10,7 @@
 
 Package: raspberrypi-sys-mods
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libcap2-bin, systemd (>= 230),
+Depends: ${shlibs:Depends}, ${misc:Depends}, libcap2-bin,
  gettext-base, procps
 Breaks: raspberrypi-ui-mods (<<1.20210706)
 Replaces: raspberrypi-ui-mods (<<1.20210706)
